docs/application/dotnet/guides/multimedia/audio.md

```diff
@@ -137,11 +137,51 @@ To query audio device information:
         ```
 
 
-        > **Note**   
-		> The initial running state of the connected device is `false`, which means the connected device is not running.
+        > **Note**
+        >
+        > The initial running state of the connected device is `false`, which means the connected device is not running.
+
+-   To utilize additional functionality for AudioDeviceType.UsbAudio and AudioDeviceIoDirection.Output device, use the [Tizen.Multimedia.AudioDevice](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.AudioDevice.html) class:
+    -   To get the device of AudioDeviceType.UsbAudio and AudioDeviceIoDirection.Output:
```

Comment:
@mijin85cho It's slightly different. In case of USB Mic, even if we try to check with AudioDeviceIoDirection.Output, we 'can not get the output device'. The codes below are to get a device that has both properties 'UsbAudio type' and 'Output direction'.

Reply:
@sangchul1011 Thanks for the comment. With the same tone with above sentence, how about below?

To get the device of UsbAudio type and Output direction:

Reply:
It seems fine to me, I'll revise it, thank you.